dataactvalidator/config/sqlrules/c12_cross_file.sql

```diff
@@ -5,26 +5,17 @@ SELECT
     ap.parent_award_id
 FROM award_procurement AS ap
 WHERE ap.submission_id = {}
-    AND COALESCE(ap.federal_action_obligation, 0) <> 0
     AND ap.piid IS NOT NULL
-    AND ((ap.parent_award_id IS NULL
-          AND NOT EXISTS (
-              SELECT 1
-              FROM award_financial AS af
-              WHERE af.submission_id = ap.submission_id
-                  AND af.piid = ap.piid)
-         )
-         OR (ap.parent_award_id IS NOT NULL
-             AND NOT EXISTS (
-                 SELECT 1
-                 FROM award_financial AS af
-                 WHERE af.submission_id = ap.submission_id
-                     AND af.piid = ap.piid
-                     AND af.parent_award_id IS NOT DISTINCT FROM ap.parent_award_id)
-         )
-    )
+    AND COALESCE(ap.federal_action_obligation, 0) <> 0
     AND NOT EXISTS (
+        SELECT 1
+        FROM award_financial AS af
+        WHERE af.submission_id = ap.submission_id
+            AND af.piid = ap.piid
+            AND af.parent_award_id IS NOT DISTINCT FROM ap.parent_award_id
+    )
+    AND NOT EXISTS(
```

Comment:
@alburde1 was kind enough to notify me that this block is no longer required to be part of this check since we're comparing D1 to C here, and `transaction_obligated_amou` is not in D1.